concepts/Common/Security Considerations/30 Export Vulnerabilities/05 CSV Injection.md

```diff
@@ -0,0 +1,191 @@
+If you use DataGrid or PivotGrid Export functionality, you should consider a threat of a CSV Injection Attack (also called formula injection attack). It involves the injection of harmful characters that act as a formula and are executed within a computer network. Cell values that begin with the **=, +, -** and **@** characters can initiate an injection attack.
```

Comment:
Is CSV the only format available for export in your `DataGrid` and `PivotGrid` products?

> If you use DataGrid or PivotGrid Export functionality

A definite article is missing here. You speak of **the** functionality available where? In `DataGrid` and `PivotGrid`.

> It involves **the** injection of harmful characters that act as a formula and are executed within a computer network.

Do we know of a specific form of such an injection in the way it always takes place? No, we do not. In the current context, it's just **an** injection.

>  Cell values that begin with the **=, +, -** and **@** characters can initiate an injection attack.

An Oxford comma is missing at the end of this enumeration.

Here's my take on this paragraph the way I see it right now. Feel free to correct me if I misinterpreted something:

```suggestion
If you export data from DataGrid or PivotGrid in CSV format, you should take a thread of a CSV Injection Attack (also known as a formula injection attack). Such attacks involve an injection of a malicious character sequence that is interpreted as a formula and executed within a computer network. Cell values that start with the **=**, **+**, **-**, and **@** characters can initiate an injection attack.
```

P.S. I also took the liberty of rephrasing the second sentence a bit for better text flow and clarity.